src/main/kotlin/io/emeraldpay/dshackle/config/context/SchedulersConfig.kt

```diff
@@ -30,6 +30,11 @@ open class SchedulersConfig {
         return makeScheduler("head-scheduler", 4, monitoringConfig)
     }
 
+    @Bean
+    open fun multistreamEventsScheduler(monitoringConfig: MonitoringConfig): Scheduler {
+        return makeScheduler("events-scheduler", 10, monitoringConfig)
```

Comment:
I think 2 is enought. 
Head scheduler is able to process ALL blocks, sub and so on with 4 threads